azure-pipelines.yml

```diff
@@ -89,6 +89,14 @@ extends:
             enableInternalSources: true
           enableTelemetry: true
           helixRepo: dotnet/templating
+          # WORKAROUND: BinSkim requires the folder exist prior to scanning.
+          templateContext:
+            sdl:
+              binskim:
+                analyzeTargetGlob: +:f|artifacts\bin\**\*.dll;+:f|artifacts\bin\**\*.exe;
+          preSteps:
+          - powershell: New-Item -ItemType Directory -Path $(Build.SourcesDirectory)/artifacts/bin -Force
+            displayName: Create artifacts/bin directory
```

Comment:
This won't do anything without this Arcade PR: https://github.com/dotnet/arcade/pull/15325

Reply:
I didn't think we were taking 15325 anymore. We needed that to do the prestep for the publish leg but by setting the templatecontext for the build itself, we only add the filter for the build. 

I thought in the SDK repo, the prestep worked for the build legs.

Reply:
Oh, derp. I thought this was a publish step. My bad.